2364/2/name_check.rb

```diff
@@ -0,0 +1,25 @@
+# Class for comparing names
+class NameCheck
```

Comment:
class names should always be verbs, if they are doing something

`NameChacker.new.run` in this case